src/sources/currencyLayer.js

```diff
@@ -0,0 +1,47 @@
+
+import config from '../config'
+import extract from '../util/extract.js'
+import load from '../util/load'
+
+const props = {
+  isProduction: config.get('env') === 'production',
+  file: './stubs/currencyLayer.json',
+  currencyLayerUrl:`${config.get('currencyLayer.token')}0&currencies=USD,AUD,CAD,PLN,MXN&format=1`,
```

Comment:
it seems you are missing the `url`, you just have `token&...` it should be something like `http://api.com?token=...`